### DISCUSSION TITLE
Importing users?

### DISCUSSION THREAD

**Discussion by osseonews** [nhost/nhost#845] [Q&A]
We were getting ready to import our existing users from another database into the users table. Our plan was to just do a bulk insert with Hasura Graphql (we already use Hasura) from the old database to Nhost hosted database. However, in your docs it says: "Never create users directly via GraphQL or database." Why is this? How else are you supposed to import alot of users if you can't use Graphql or the database to do an import? It's not really practical to insert every users one-by-one with a call to the signup flow.

Incidentally, how do we prevent users that we import from getting a verification email on an import, as they are already verified. Is there some boolean field to set as true upon import? I didn't see in the docs what fields are available on the users table, other than email and password.

**Comment by elitan** [comment]
If you'r importing users, you can disregard this:

> "Never create users directly via GraphQL or database."

I'll update the docs to make the answer more nuanced. 

> Incidentally, how do we prevent users that we import from getting a verification email on an import, as they are already verified. 

My creating the users directly via GraphQL, as you've planned. 

> Is there some boolean field to set as true upon import?

Set the `emailVerified` field to `true` for the user.
